### PR TITLE
Add ExcludeLink and Priority as Email options

### LINKED ISSUE #85: New-RsSubscription does not support Email parameters IncludeLink and Priority
*Feature*

This seems easy enough.

Version 0.0.2.9

### PR DESCRIPTION
**Pull request by jzabroski**
Fixes #85.

Changes proposed in this pull request:
 - Add ExcludeLink and Priority as valid Email parameters
 - Enforce valid set of Priorty options
 - Enforce default value of NORMAL for Priority

How to test this code:
 - I'm open to suggestions

Has been tested on (remove any that don't apply):
 - Powershell 5 and above
 - Windows 10 and above
 - SQL Server 2016 and above

### CODE REVIEW

**Comment by jzabroski** [comment]
I will add support for ReplyTo and Comments fields separately. I don't need those right now but as I see gaps I will help fill.

**Comment by jtarquino** [comment]
For testing check https://github.com/Microsoft/ReportingServicesTools/blob/master/Tests/CatalogItems/New-RsSubscription.Tests.ps1 and you can extend it for your new properties

File: ReportingServicesTools/Functions/CatalogItems/New-RsSubscription.ps1 [hunk 1]
```diff
@@ -59,6 +59,12 @@ function New-RsSubscription {
         .PARAMETER RenderFormat
             Specify the output format of the report. Must be one of 'PDF','MHTML','IMAGE','CSV','XML','EXCELOPENXML','ATOM','PPTX','WORDOPENXML'
         
+        .PARAMETER ExcludeLink
+            Use with -Destination 'Email' to exclude a link back to the report from the email.
+        
+        .PARAMTER Priority
+            Use with -Destination 'Email' to set the priority with which the e-mail message is sent. Valid values are LOW, NORMAL, and HIGH. The default value is NORMAL.
+            
         .EXAMPLE
```
Comment (parthsha) [inline ReportingServicesTools/Functions/CatalogItems/New-RsSubscription.ps1]: Please create an example with these options.

File: ReportingServicesTools/Functions/CatalogItems/New-RsSubscription.ps1 [hunk 2]
```diff
@@ -59,6 +59,12 @@ function New-RsSubscription {
         .PARAMETER RenderFormat
             Specify the output format of the report. Must be one of 'PDF','MHTML','IMAGE','CSV','XML','EXCELOPENXML','ATOM','PPTX','WORDOPENXML'
         
+        .PARAMETER ExcludeLink
+            Use with -Destination 'Email' to exclude a link back to the report from the email.
+        
+        .PARAMTER Priority
```
Comment (parthsha) [inline ReportingServicesTools/Functions/CatalogItems/New-RsSubscription.ps1]: Please update ".PARMTER" to ".PARAMETER"

**Comment by jzabroski** [comment]
OK, I'll have this done either tonight or tomorrow.

**Comment by parthsha** [comment]
Hey John, have you had a chance to look into this?

**Comment by jzabroski** [comment]
Hi @parthsha  - sorry for the hold up. Heading on vacation in a few days and had to deal with some visa stuff before I head out of the country. Think I will get to this today.

**Comment by jzabroski** [comment]
Sorry I haven't written tests. I am now on vacation but will live to finish
this first chance I get

**Pull request merged**